### ISSUE TITLE
Panels that pop up as a result of an action aren't read by screen readers

### ISSUE THREAD

**Issue by allinox** [Altinn/altinn-access-management-frontend#365]
### Description

Consider the case where a user is on the Choose org page and har one previously used org.
When adding this org, the panel about searching for new orgs appears but goes unnoticed by screen readers.

This is the case with all these alerts (new org and org not found) and the lack of screenreader visibility in this change breaks WCAG.

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/d0662e62-5e43-4889-a9b7-113d6abb1d46)

**Comment by allinox** [comment]
@Albertlarsen 

What do you think? Should we prioritize doing this one now or should we put it back into no-status?

**Comment by rtafj** [comment]
@allinox - vet du om denne ble fikset med https://github.com/Altinn/altinn-access-management-frontend/pull/528? Hvis ja - lukker du denne da?

**Comment by allinox** [comment]
@rtafj 
Nei, den PRen ble lukket uten å merges i main for den løste ikke faktisk problemet.
Tasken ble satt tilbake til NoStatus så den kan ikke lukkes enda

**Comment by allinox** [comment]
This issue is now baked into #789
